docs/doxygen-user/multi-user/installMultiUser.dox

```diff
@@ -0,0 +1,40 @@
+/*! \page install_multiuser_page Setting Up Multi-user Cluster
+
+\section multiuser_install_overview Overview
+
+Autopsy can be setup to work in an environment where multiple users on different computers can have the same case open at the same time.  To set up this type of environment, you will need to configure additional (free and open source) network-based services.  
+
+The basic concept is that you'll have a central:
+- Database 
+- Keyword search index
+- Storage
+
+Each Autopsy client will then use those shared resources instead of the embedded versions that are used for single-user cases.  
+
+\image html multi-user-network.png
+
+
+\section multiuser_install_install Cluster Installation and Configuration
+
+Let's now step through the process of setting up an Autopsy cluster. 
+
+Step 1: \ref multiuser_system
```

Comment:
I personally would prefer the link to go to install_multiuser_systems_page instead of the first section. The way it is makes me think that there's more data above where the link went to, but it's just the page title. There are a few like this.

Reply:
Fixed and renamed the anchor names to consistently mention they were pages.